ldap/servers/plugins/uiduniq/uid.c

```diff
@@ -925,7 +925,7 @@ findSubtreeAndSearch(Slapi_DN *parentDN, const char **attrNames, Slapi_Attr *att
            * to have the attribute already.
            */
             result = search(curpar, attrNames, attr, values, requiredObjectClass,
-                            target, excludes);
```

Comment:
This is something that I do not understand. uiduniqness should check if the uid value already exist in the parent of the destinationSDN, not in the parent of the sourceSDN.
IMHO it should be slapi_sdn_get_parent(destinationSDN, curpar);